### PR TITLE
rdgo: build crio

### PR DESCRIPTION
**Pull request by ashcrow**
This will build the latest commits off the 1.10 branch.

### CODE REVIEW

File: overlay.yml [hunk 1]
```diff
@@ -21,5 +18,10 @@ components:
     override-version: "1.13.1"
     branch: docker-1.13.1-rhel
 
+  # NOTE: master=1.11.0 which requires a different spec
```
Comment (jlebon) [inline overlay.yml]: Ahh OK. Hmm, yeah we'll have to adapt the spec file so we can track git master.

File: overlay.yml [hunk 2]
```diff
@@ -21,5 +18,10 @@ components:
...
+  - src: github:kubernetes-incubator/cri-o
+    override-version: "1.10.1"
```
Comment (jlebon) [inline overlay.yml]: What happens without this? I see 1.10.1 tagged on the `release-1.10` branch. I thought rdgo would pick up on that.

Comment (ashcrow) [reply overlay.yml]: The result I got was 1.9 rpms with 1.10.1 content.

**Comment by peterbaouoft** [comment]
FWIW, WFM :). 
```
[root@a20e073781a4 cri-o-1.10.1-095e88ca5847456b497ad2616fdc63c0debd3d49.556df46c5207238a3ee196220bd13f975551467e]# ls | grep rpm
conmon-1.10.1-095e88ca5847456b497ad2616fdc63c0debd3d49.556df46c5207238a3ee196220bd13f975551467e.fc28.x86_64.rpm
conmon-debuginfo-1.10.1-095e88ca5847456b497ad2616fdc63c0debd3d49.556df46c5207238a3ee196220bd13f975551467e.fc28.x86_64.rpm
cri-o-1.10.1-095e88ca5847456b497ad2616fdc63c0debd3d49.556df46c5207238a3ee196220bd13f975551467e.fc28.src.rpm
cri-o-1.10.1-095e88ca5847456b497ad2616fdc63c0debd3d49.556df46c5207238a3ee196220bd13f975551467e.fc28.x86_64.rpm
cri-o-debuginfo-1.10.1-095e88ca5847456b497ad2616fdc63c0debd3d49.556df46c5207238a3ee196220bd13f975551467e.fc28.x86_64.rpm
cri-o-debugsource-1.10.1-095e88ca5847456b497ad2616fdc63c0debd3d49.556df46c5207238a3ee196220bd13f975551467e.fc28.x86_64.rpm
cri-o-integration-tests-1.10.1-095e88ca5847456b497ad2616fdc63c0debd3d49.556df46c5207238a3ee196220bd13f975551467e.fc28.x86_64.rpm
cri-o-integration-tests-debuginfo-1.10.1-095e88ca5847456b497ad2616fdc63c0debd3d49.556df46c5207238a3ee196220bd13f975551467e.fc28.x86_64.rpm
srpm
```

**Pull request merged**